deps/jerry/jerry-core/profiles/ShadowNode.profile

```diff
@@ -0,0 +1,2 @@
+CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
```

Comment:
To keep the consistent with other profiles, suggestion rename: `shadow-node.profile`.

BTW, could you please also move the this profile to ShadowNode root directory, JerryScript supports the absolute path at [deps/jerry/jerry-core/CMakeLists.txt#L236-L247](https://github.com/Rokid/ShadowNode/blob/master/deps/jerry/jerry-core/CMakeLists.txt#L236-L247).